images/virt-artifact/patches/017-fix-vmi-subresource-url.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/pkg/virt-operator/resource/generate/rbac/controller.go b/pkg/virt-operator/resource/generate/rbac/controller.go
```

Comment:
we can opt out of "VolumeRequests" in kvvm and using only in kvvmi. Then our subresource "addVolume" should mutate kvvm spec and delegate request to virt-api. if we do it , we won't need this patch.